libs/core/src/components/pds-chip/pds-chip.scss

```diff
@@ -1,82 +1,78 @@
-  --background-accent-dot: var(--pine-color-purple-300);
-  --background-accent-interactive: var(--pine-color-purple-200);
-  --background-success: var(--pine-color-green-100);
-  --background-success-dot: var(--pine-color-green-300);
-  --background-success-interactive: var(--pine-color-green-200);
-  --background-warning: var(--pine-color-yellow-100);
-  --background-warning-dot: var(--pine-color-yellow-300);
-  --background-warning-interactive: var(--pine-color-yellow-200);
-  --box-shadow-focus: 0 0 0 2px var(--pine-color-primary-200);
-
-  --border-radius: var(--pine-border-radius-lg);
-  --font-size: var(--pine-font-size-body-sm);
-  --font-size-lg: var(--pine-font-size-body);
+  --box-shadow-focus: 0 0 0 2px var(--pine-color-blue-200);
+  --border-radius: var(--pine-border-radius-200);
+
+  --color-accent: var(--pine-color-purple-100);
+  --color-accent-dot: var(--pine-color-purple-300);
+  --color-accent-interactive: var(--pine-color-purple-200);
+  --color-danger: var(--pine-color-red-100);
+  --color-danger-dot: var(--pine-color-red-300);
+  --color-danger-interactive: var(--pine-color-red-200);
+  --color-info: var(--pine-color-blue-100);
+  --color-info-dot: var(--pine-color-blue-300);
+  --color-info-interactive: var(--pine-color-blue-200);
+  --color-neutral: var(--pine-color-grey-300);
+  --color-neutral-dot: var(--pine-color-charcoal-100);
+  --color-neutral-interactive: var(--pine-color-grey-400);
+  --color-success: var(--pine-color-green-100);
+  --color-success-dot: var(--pine-color-green-300);
+  --color-success-interactive: var(--pine-color-green-200);
+  --color-warning: var(--pine-color-yellow-100);
+  --color-warning-dot: var(--pine-color-yellow-300);
+  --color-warning-interactive: var(--pine-color-yellow-200);
+
+  --font-size-sm: var(--pine-font-size-087);
+  --font-size-body: var(--pine-font-size-100);
   --font-weight: var(--pine-font-weight-medium);
-  --line-height: var(--pine-line-height-sm);
-  --padding-block-sm: calc(var(--pine-spacing-xxs) / 2);
-  --padding-block: var(--pine-spacing-xxs);
-  --padding-block-lg: 6px;
-  --padding-inline-sm: calc(var(--pine-spacing-xxs) / 2);
-  --padding-inline: 10px;
-  --padding-inline-lg: calc(var(--pine-spacing-md) / 2);
-  --padding-inline-xl: 14px;
-  --margin-block-xs: calc(var(--pine-spacing-xxs) / 2);
-  --margin-inline-sm: calc(var(--pine-spacing-xs) / 2);
-  --margin-inline-xs: calc(var(--pine-spacing-xxs) / 2);
-  --margin-inline: 4px;
-  --margin-inline-lg: var(--pine-spacing-xs);
-  --margin-inline-xl: calc(var(--pine-spacing-md) / 2);
+  --line-height: var(--pine-line-height-100);
+
+  --spacing-xxs: calc(var(--pine-spacing-050) / 2);
+  --spacing-xs: var(--pine-spacing-050);
+  --spacing-sm: calc(var(--pine-spacing-150) / 2);
+  --spacing-md: var(--pine-spacing-100);
+  --spacing-lg: calc(var(--pine-spacing-250) / 2);
+  --spacing-xl: var(--pine-spacing-150);
+  --spacing-xxl: calc(var(--pine-spacing-350) / 2);
```

Comment:
Could we add context to these tokens as well? How they're being applied? Margin, padding, etc?

Reply:
these are used for both padding/margins. what else could i use?

Reply:
I was thinking making a distinction in usage between block and inline if it applies.

Reply:
@anechol This is the area where I am most confused I guess. If I have to break them out into each possible spacing type inline, block, start, end, etc... we end up with a lot of repeating values and one-off variables. To me that would defeat the purpose of using the variables and would be better to just add them separately to each selector instead.

I may be misunderstanding but let me know.